CHANGELOG.md

```diff
@@ -9,14 +9,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 ### Added
 
 - Cinnamon Sword
+- Intellibeam Lazerstation
+- Unreadable tag
 
 ### Fixed
 
 - Basilisks can now despawn again
 
 ### Contributors for this release
 
-- Vivian-Zane, rose_bushes_
+- Vivian-Zane, rose_bushes_, Doro
```

Comment:
Make sure to include blankMushroom

Reply:
done

Reply:
Looks like that change has been lost since then?